utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns a random object from the array of colors to user
+ * @param array {array} : array containing objects
+ * @returns random Index number : index between the range 0 to array.length
+ */
+const getRandomIndex = (arrayLength = 10) => {
+  return Math.floor(Math.random() * arrayLength);
+};
```

Comment:
this will return 1 less than array length as we are using math.floor. do we need even the upper bound i.e 10 here. if yes we should use `math.ceil`

Reply:
if we have an array of length 10, then we can only index it upto arr[9], thats why  `math.floor`